static/styles/left_sidebar.css

```diff
@@ -316,8 +316,10 @@ li.top_left_recent_topics {
 .top_left_starred_messages .count {
     background-color: transparent;
     color: inherit;
-    padding: 2px 1px 0 3px;
+    padding: 0 3px 0 3px;
     border-color: hsl(105, 2%, 50%);
+    border: 1.5px solid hsl(0, 0%, 66%);
```

Comment:
If you are specifying the border color in the `border` line, can't we eliminate `border-color` above it?

Reply:
Yes, we can  eliminate it as it doesn't effect any of this change. So i will eliminate that.